### PR TITLE
Add editorconfig maven plugin

### PR DESCRIPTION
**Pull request by dr0i**
See #1997.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -247,7 +247,40 @@
...
+                        <execution>
+                            <id>check</id>
+                            <phase>verify</phase>
+                            <goals>
+                                <goal>check</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                    <configuration>
+                        <excludes>
```
Comment (TobiasNx) [inline pom.xml:265]: Do we need excludes, if it seems to include only the `includes`?

File: pom.xml [hunk 2]
```diff
@@ -247,7 +247,40 @@
...
+                            <exclude>vocabs</exclude>
+                            <exclude>lookup-tables</exclude>
+                            <exclude>**/*.tsv</exclude>
```
Comment (TobiasNx) [inline pom.xml:268]: Shouldn't we include the tsv too? It has spotted a trailing whitespace which created wrong links.  Isn't that a good thing? See: https://github.com/hbz/lobid-resources/pull/1997/commits/acb15f467217d33bb2d200a17ae7cc89846d4920

File: pom.xml [hunk 3]
```diff
@@ -247,7 +247,40 @@
...
+                            <exclude>web/public/stylesheets/bootstrap.min.css</exclude>
+                            <exclude>web/public/stylesheets/font-awesome.min.css</exclude>
+                            <exclude>web/public/stylesheets/jquery-ui.min.css</exclude>
+                            <exclude>web/public/stylesheets/leaflet.css</exclude>
+                            <exclude>web/public/stylesheets/octicons.css</exclude>
+                        </excludes>
+                        <includes>
+                            <include>pom.xml</include>
+                            <include>.editorconfig</include>
+                            <include>src/main/</include>
```
Comment (TobiasNx) [inline pom.xml:278]: Does this needs an asterisk at the end?

**Comment by TobiasNx** [comment]
This is 13 commits behind master, perhaps this needs to be rebased before solving the conflicts.

**Comment by dr0i** [comment]
Replaced by #2007. Closing.

**Pull request closed without merging**